File: 2.0/python/contacts/getContactListItems.py
from callfire.client import CallfireClient

client = CallfireClient('api-login', 'api-password')
response = client.contacts.getContactListItems(
    id=379506003,
    # search offset
    offset=0,
    # return 10 items per request
    limit=10,
    # return only specific fields
    fields='items(firstName,lastName,workPhone)'
).result()

# see sample JSON response for this API
# on 'curl' samples tab
print(response)

File: 2.0/php/contacts/getContactListItems.php
<?php

class ApiClientSample {

    public static function main() {
        $client = \CallFire\Api\DocumentedClient::createClient("login", "password");
        $request = $client->getContactListItems();
        $request->getOperationConfig()->setPathParameters(array("id" => 379506003));
        $request->getOperationConfig()->setQueryParameters(array("offset" => 0,
                                                                 "limit" => 10,
                                                                 "fields" => "items(firstName,lastName,workPhone)"));
        $result = $client->request($request);
        $json = json_decode($result->getBody());
    }
}

ApiClientSample::main();
